core/drivers/crypto/caam/caam_rng.c

```diff
@@ -556,10 +554,8 @@ enum caam_status caam_rng_init(vaddr_t ctrl_addr)
 		retstatus = caam_rng_instantiation();
 	}
 
-#ifdef CFG_NXP_CAAM_RNG_DRV
```

Comment:
`CFG_NXP_CAAM_RNG_DRV` enables or not the request of CAAM RNG data at boot or runtime. I cannot accept that modification.

Reply:
I can use CFG_WITH_SOFTWARE_PRNG instead. Would that be ok?

Reply:
That would be better

Reply:
Done just that.